apps/src/aiTutor/views/teacherDashboard/AccessControls.tsx

```diff
@@ -1,20 +1,18 @@
 import React, {useEffect, useState} from 'react';
 import {fetchStudents} from '@cdo/apps/aiTutor/accessControlsApi';
-import StudentAccessToggle from './studentAccessToggle';
-import style from './chat-messages-table.module.scss';
+import StudentAccessToggle from './StudentAccessToggle';
+import style from './interactions-table.module.scss';
 import {StudentAccessData} from '@cdo/apps/aiTutor/types';
 
 /**
  * Renders toggles to control student access to AI Tutor.
  */
 
-interface AITutorTeacherControlsProps {
+interface AccessControlsProps {
```

Comment:
I initially started renaming things because I was having trouble with git not detecting case changes in filenames when I moved from camelCase to PascalCase on these components (I had to do a combination of clearing my VSCode cache, modifying my git config, and using `git mv` to get it to pick all of them up). But I left the simpler names in place. Now that these are all scoped to the `src/aiTutor` directory, it doesn't feel like they all need to be prepended with `AITutor`.